dotCMS/src/main/java/com/dotcms/saml/SamlValidator.java

```diff
@@ -0,0 +1,50 @@
+package com.dotcms.saml;
+
+import com.dotcms.api.system.event.message.MessageSeverity;
+import com.dotcms.api.system.event.message.MessageType;
+import com.dotcms.api.system.event.message.SystemMessageEventUtil;
+import com.dotcms.api.system.event.message.builder.SystemMessage;
+import com.dotcms.api.system.event.message.builder.SystemMessageBuilder;
+import com.dotmarketing.util.DateUtil;
+import com.dotmarketing.util.UtilMethods;
+import com.dotmarketing.util.XMLUtils;
+import com.liferay.portal.language.LanguageUtil;
+import io.vavr.control.Try;
+
+import java.util.Arrays;
+
+public class SamlValidator {
```

Comment:
Codacy found an issue: [All methods are static.  Consider using a utility class instead. Alternatively, you could add a private constructor or make the class abstract to silence this warning.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5707914)